user_manual/files/webgui/custom_groups.rst

```diff
@@ -10,11 +10,9 @@ To address that, as of ownCloud 10.0, you can now create your own groups on-the-
 Here's how to use it. 
 
 .. note::
-	Depending on your custom groups setting, configured by the ownCloud admin, custom groups may behave differently
-	
-	- Creating or renaming a custom group using an existing display name will show an error
-	- A config switch (checkbox) makes it possible to allow multiple custom groups with the same name
-	- Custom group creation can be limited to the ownCloud **group admins**
+	Depending on your Custom Groups setting, configured by the ownCloud admin, Custom Groups may behave differently
+	- Creating or renaming a Custom Group using an existing name of another Custom Group can be allowed or not depending on administrative settings.
+	- Custom Group creation can be limited to the ownCloud **group admins**
```

Comment:
@settermjd pls correct me if my Engish is bad, but isnt it 
``limited to the ownCloud`` --> ``limited to ownCloud``

Reply:
the "the" is for the group admins, not the the "ownCloud"

But yeah, I always put a "the" to much, because it's better then a "the" to few :)

Reply:
``limited to ownCloud`` would be correct when ``ownCloud`` is the noun.
but when used as an adjective:
``limited to the ownCloud server``
``limited to the ownCloud administrators``
the "the" depends on the following noun.

Reply:
I see that I will have to go for a training to your location 😄

Reply:
for just a small training fee of 49,99 Euro ;)